skip-tests.json

```diff
@@ -49,9 +51,7 @@
         }
     },
     "binance": {
```

Comment:
@ttodua u sure the php proxy is working properly? 
![image](https://github.com/ccxt/ccxt/assets/43336371/46d2969f-1004-4d33-99c1-b091cb6e1272)